### DISCUSSION TITLE
Strange behaviour of the discord integration

### DISCUSSION THREAD

**Discussion by ghost** [Playabl-io/playabl#263] [General]
Posting it here because I think this is me not understanding how things work rather than a bug :)

We use integrations on our community to receive notifications for new games on discord and on a few other platforms. Recently, games posted on Playabl by one of our users are not appearing on Discord, but work fine with the other integrations for other platforms. 

To reiterate, this is a problem only for games from that one user and only with the integration with Discord. 

I cannot see what we are doing wrong and it's doing my head in a bit :)

Any advice on what can be going on? Thanks!

**Comment by jongrim** [comment]
Hey! Sorry for the late response. That does sound like a bug to me, as the integration should fire for any game posted by anyone in the community. There's something making it error out though, so it'll take some research. Can you give me some details and I'll investigate. A link to one of the recent games that didn't post is a good starting point.

**Reply by ghost** [reply]
Thank you. These two did not work on discord, but fine with integrations with telegram and mastodon:
https://app.playabl.io/games/480/overview
https://app.playabl.io/games/493/overview

**Comment by jongrim** [comment]
Aha, I see the issue. [There is a limit to the number of characters allowed in a field](https://discord.com/safety/using-webhooks-and-embeds). For description - where the game description gets added - this is 2048 characters, so I need to update the code to trim the text. I should be able to fix that quickly and have it out in the next day or two. Thanks for reporting!

**Reply by ghost** [reply]
Wonderful, thanks for looking into this!

**Reply by jongrim** [reply]
I've deployed the updated code. I unfortunately don't have a great way to test it without posting a game to someone's discord, but the change was very simple so I have a high confidence rating. Let me know if you see any more issues!

**Reply by ghost** [reply]
I think there's still something not right. 
This game did not get posted on discord and also not on telegram or mastodon
https://app.playabl.io/games/493/overview
This is from the same user as the other two posted above.

Other games from other users posted after this had no problems with any of the integrations.
